.github/workflows/main.yml

```diff
@@ -8,21 +8,14 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Set up JDK 19
+        with:
```

Comment:
We'd probably want to keep this as JDK 19 as it'll keep parity with what we're also running locally.